tests/integration_tests/test_llm.py

```diff
@@ -809,7 +812,7 @@ def test_llm_lora_finetuning_merge_and_unload_4_bit_quantization_not_supported(l
             [
                 "README.md",
                 "adapter_config.json",
-                "adapter_model.bin",
+                "adapter_model.safetensors",
```

Comment:
@arnavgarg1 I see what happened; these changes back and forth from bin to safetensors was in response to PEFT having bugs due to changes in the point releases.

Reply:
Ah! I see, good to have that context @alexsherstinsky!